### DISCUSSION TITLE
ISV Success program, Github Enterprise Cloud Subscription

### DISCUSSION THREAD

**Discussion by alexgenne** [community/community#54354] [Enterprise]
Hi,
I recently created a Github account for our company. With this account I then created an enterprise and an organization.

The trial period will now expire in two days.

Our company has benefits from the ISV Success program. The benefit is: "Github Enterprise Cloud Subscription"
I have redeemed the benefits/discount with my account and would like to know how I can apply these benefits to the existing Enterprise Account.

The discount shows up in my personal account but not in the enterprise account. Can the discount be somehow transferred to the Enterprise Account?

Thanks in advance

**Comment by queenofcorgis** [comment]
Hi @alexgenne ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by alexgenne** [reply]
Hi @queenofcorgis, I already opened a support ticket. This was the answer: 
"Your ticket has been automatically routed to the Enterprise Support team but the Security & Revenue Support team is better suited to help out, so we're forwarding your ticket to them."
From that point I never hered anything.

I attempted to create another one, but it was linked to the existing one and closed immidiatly.

**Comment by hbunjes** [comment]
Have you ever received an answer? We have the same problem.

**Comment by EnkiNijlTaluvaar** [comment]
Same here
